operator/api/v1alpha1/offering_types.go

```diff
@@ -20,38 +20,42 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// EDIT THIS FILE!  THIS IS SCAFFOLDING FOR YOU TO OWN!
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
-
-// OfferingSpec defines the desired state of Offering
+// OfferingSpec defines the Service Offering spec.
 type OfferingSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
-
-	// Foo is an example field of Offering. Edit Offering_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+	// The name of the Provider this Offering belongs to. The Offering controller sets ownership to
+	// the Provider specified here.
```

Comment:
```suggestion
	// The name of the Provider this Offering belongs to. This offering will automatically be
	// deleted when the provider specified here is deleted.
```

Also, it's probably a good idea to insert a blank line between fields.

Reply:
Fixed with https://github.com/SUSE/metabroker/pull/6/commits/a35303e8893e92dbf15e1dffa8e3660b0aa1d254.

Reply:
Force-pushed https://github.com/SUSE/metabroker/pull/6/commits/6a882e28f1a7bb3fbd919e15ac27d459f62453cb to include the generated files.